docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -1,843 +0,0 @@
-
-===== Example: Assign index to a specific node and update replica settings
-
-This example updates the index to have one replica per shard and be allocated
-to nodes with a `box_type` attribute of "cold".
-
-[source,console]
---------------------------------------------------
-PUT _ilm/policy/my_policy
-{
-  "policy": {
-    "phases": {
-      "warm": {
-        "actions": {
-          "allocate" : {
-            "number_of_replicas": 1,
-            "require" : {
-              "box_type": "cold"
-            }
-        }
-        }
-      }
-    }
-  }
-}
---------------------------------------------------
-
-[[ilm-wait-for-snapshot-action]]
-==== Wait For Snapshot
-
-Phases allowed: delete.
-
-The Wait For Snapshot Action waits for defined SLM policy to be executed to ensure that snapshot of index exists before
-deletion.
-
-[[ilm-wait-for-snapshot-options]]
-.Wait For Snapshot
-[options="header"]
-|======
-| Name                 | Required  | Default             | Description
-| `policy`             | yes       | -                   | SLM policy name that this action should wait for
-|======
-
-[source,console]
---------------------------------------------------
-PUT _ilm/policy/my_policy
-{
-  "policy": {
-    "phases": {
-      "delete": {
-        "actions": {
-          "wait_for_snapshot" : {
-            "policy": "slm-policy-name"
-          }
-        }
-      }
-    }
-  }
-}
---------------------------------------------------
-
-[[ilm-delete-action]]
-==== Delete
-
-Phases allowed: delete.
-
-The Delete Action does just that, it deletes the index.
-
-[[ilm-delete-action-options]]
-.Delete
-[options="header"]
-|======
-| Name                         | Required  | Default             | Description
-| `delete_searchable_snapshot` | no        | true                | Deletes the searchable snapshot created in the cold phase, if a
-                                                                   snapshot was created (ie. if the <<ilm-searchable-snapshot-action,
-                                                                   Searchable Snapshot Action>> was used in the cold phase)
-|======
-
-[source,console]
---------------------------------------------------
-PUT _ilm/policy/my_policy
-{
-  "policy": {
-    "phases": {
-      "delete": {
-        "actions": {
-          "delete" : { }
-        }
-      }
-    }
-  }
-}
---------------------------------------------------
-
-[[ilm-forcemerge-action]]
-==== Force Merge
-
-Phases allowed: hot, warm.
-
-NOTE: Index will be be made read-only when this action is run
-(see: <<dynamic-index-settings,index.blocks.write>>)
-
-NOTE: The `forcemerge` action is best effort. It might happen that some of the
-shards are relocating, in which case they will not be merged.
```

Comment:
@debadair I think this got lost in the merge. I think we should re-add it to `ilm-forcemerge.asciidoc`